packages/classic-test-app/testem.js

```diff
@@ -1,7 +1,6 @@
-/* eslint-env node */
+'use strict';
 
 module.exports = {
-  framework: 'mocha',
```

Comment:
Is this not needed anymore?

Reply:
This is what `ember init` generates but it seems fine since the [testem docs](https://github.com/testem/testem/blob/master/docs/config_file.md#common-configuration-options) say:

> **framework** - the test framework that you are using, in the browser, in the case that you are not also using the test_page option.